astroquery/sdss/core.py

```diff
@@ -144,7 +144,10 @@ def query_crossid_async(self, coordinates, obj_names=None,
 
         sql_query += ',dbo.fPhotoTypeN(p.type) as type \
                       FROM #upload u JOIN #x x ON x.up_id = u.up_id \
-                      JOIN PhotoObjAll p ON p.objID = x.objID ORDER BY x.up_id'
+                      JOIN PhotoObjAll p ON p.objID = x.objID '
+        if specobj_fields:
```

Comment:
Can you add a test when `specobj_fields` is set? I think that would be very useful to have.

Reply:
I wrote a new use case similar to the following written without spec in commit fb84a65

https://github.com/astropy/astroquery/blob/564406770eaf332f156ca0eba28ee561d041925a/astroquery/sdss/tests/test_sdss_remote.py#L169-L176